remoteit/desktop
# [DESK-743] Fonts update and scanEnabled feature activated in new ui

### Description


Added the scan feature control
Updated fontawesome to include the 'radar' icon.

### Test plan



1. Sign in to an AWS device
2. Go to the device on the device list
3. Click edit device
4. You should not see the scan icon on the page.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-743>
